apps/microsoft-teams/app-actions/src/actions/send-test.spec.ts

```diff
@@ -29,8 +39,11 @@ describe('sendTestNotification.handler', () => {
     context = makeMockAppActionCallContext(cmaClientMockResponses, cmaRequestStub);
   });
 
-  it('returns the ok result', async () => {
-    const result = await handler({ channel: 'foo' }, context);
-    expect(result).to.have.property('ok', true);
+  it('calls the cma to get the tenant id from app installation params', async () => {
```

Comment:
In addition to the test you have here (which verifies the side effect of calling the handler) I always find it nice to include a test that verifies the output (in this case the JSON blog with the ok: true). Would be nice to have this test here.

Reply:
Yes 💯! I will address that in a follow up